extensions/amp-list/amp-list.md

```diff
@@ -282,33 +282,62 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 If `binding` attribute is not provided, default is `always`.
 
 ## Experimental: Infinite Scroll (amp-list-load-more)
-We've introduced an experiment called `amp-list-load-more` as an implementation for pagination and infinite scroll in amp-list. This is an experimental feature, and final APIs may change.
+We've introduced the `amp-list-load-more` experiment as an implementation for pagination and infinite scroll in `amp-list`. This is an experimental feature, and final APIs may change.
 
 ### Attributes
 #### load-more (mandatory)
-Adding this attribute will allow amp-list (with no value) to show a “load-more" button at the end of the amp-list. The value of this attribute can be set to “auto" to trigger automatic loading more elements three viewports down for an infinite scroll effect.
+Adding this attribute allows amp-list (with no value) to show a “load-more" button at the end of the amp-list. The value of this attribute can be set to “auto" to trigger automatic loading more elements three viewports down for an infinite scroll effect.
 
-#### load-more-bookmark (mandatory)
-This attribute specifies an attribute in the returned data that will give the url of the next items to load. E.g. In the following sample payload, we would specify `load-more-bookmark="next"`.
+#### load-more-bookmark (optional)
+This attribute specifies an attribute in the returned data that will give the url of the next items to load. If this attribute is not specified, `amp-list` expects the json payload to have the `load-more-src` field, which corresponds to the next url to load. In the case where this field is called something else, you can specify the name of that field via the `load-more-bookmark` field.E.g. In the following sample payload, we would specify `load-more-bookmark="next"`.
 
 ```
-{ "items": [], "next": "https://url.to.load" }
+{ "items": [...], "next": "https://url.to.load" }
 ```
 
-### Additional children of `<amp-list>`
-`<amp-list>` with the `load-more` attribute expects the following additional child elements. All of these elements are templated. (Note that UX defaults for these are on the roadmap, thus it will not always be necessary to include these elements.)
+### Customizing load-more elements
+`<amp-list>` with the `load-more` attribute contains these UI elements: a load-more button, a loader, a load-failed element, and optionally an end-cap marking the end of the list. These elements can be customized by providing elements as children of `amp-list` with the following attributes:
```

Comment:
"an be customized by providing elements as children "
Is there a list of elements that can be used or a list of elements that cannot be used? We should include whichever list is shorter.

Reply:
I may defer to @andrewwatterson for this. I think ideally we want them to be buttons? There are currently no restrictions on this, but maybe we should restrict this to a whitelist or even buttons  (divs containing buttons) only?

Reply:
Do we tend to aggressively whitelist/limit things like this? I'd probably prefer to leave it more open if folks are comfortable with that. `div`s and `button`s should get us most of the way, but I can imagine someone using an `img` for the loader, and there are probably other legitimate custom use cases I'm not thinking of here.

Reply:
Per our offline discussion, I think it's better to initially be conservative here because: 
1. It's always easier to relax constraints than to add them after launch. 
2. Allowing anything opens a pandora's box for abusing this to create bad UX e.g. insert a ton of videos and ads.